src/apronPrivPrecCompare.ml

```diff
@@ -0,0 +1 @@
+module B = PrePrivPrecCompare.ComparePrec (PrivPrecCompareUtil.ApronUtil)
```

Comment:
It's kind of weird that just applying this functor will already do something imperative. I think there should be an actual function to be explicitly called from these, e.g. `B.main ()`.